src/models/TechRecordsDAO.ts

```diff
@@ -96,12 +97,11 @@ class TechRecordsDAO {
   }
 
   public updateSingle(techRecord: ITechRecordWrapper) {
-    const partialVin = techRecord.vin.substr(techRecord.vin.length - 6);
-
+    techRecord.partialVin = populatePartialVin(techRecord.vin);
     const query = {
       TableName: this.tableName,
       Key: {
-        partialVin,
+        partialVin: techRecord.partialVin,
```

Comment:
Good for now, but just a note for posterity that primary key changes will be coming through soon.